types/uint64.go

```diff
@@ -1,12 +1,11 @@
 package types
 
 import (
-	"encoding/base64"
-	"strings"
-
 	"gx/ipfs/QmSKyB5faguXT4NqbrXpnRXqaVj5DhSm7x9BtzFydBY1UK/go-leb128"
 	cbor "gx/ipfs/QmcZLyosDwMKdB6NLRsiss9HXzDPhVhhRtPy67JFKTDQDX/go-ipld-cbor"
 	"gx/ipfs/QmdBzoMxsBpojBfN1cv5GnKtB7sfYBMoLH7p9qSyEVYXcu/refmt/obj/atlas"
+	"strconv"
+	"strings"
```

Comment:
Please revert this reordering. Our conventions are documented in https://github.com/filecoin-project/go-filecoin/blob/master/CONTRIBUTING.md#conventions-and-style